.github/workflows/build.yml

```diff
@@ -69,8 +75,10 @@ jobs:
   lint:
     name: ${{ matrix.toxenv }}
     runs-on: ubuntu-latest
+    continue-on-error: true
```

Comment:
I don't think this is right.  I want the workflow to fail when one of the linters complains.

Reply:
So if a linter fails, the build jobs should be cancelled?

I haven't used it before, but I got the impression that there is a way using job dependencies to run the linters first and only start the build jobs if the linters pass. Is that what you're going for?

Reply:
Oh, is that how it works?  GitHub Actions are very confusing to me.

Reply:
I'm not 100% sure, but my impression from watching my PRs lately is that if continue-on-error is false then if a linter job finishes first and fails then all of the rest of the running or scheduled jobs are canceled. I can understand that being GitHub's preference, since it means fewer resources used on a PR that's not likely to be merged. I was trying to configure things so that all of the jobs would run, regardless of whether other jobs fail. I could try to set up the linter jobs to run first, and only if they pass let the build jobs run, if that seems like a better compromise?

Reply:
I believe what you describe is what this PR sets up. It's definitely what I was _trying_ to set up. :-)